lib/Cro/Message.pm6

```diff
@@ -2,6 +2,26 @@
 # application is processing. It might be a message from a message queue, a
 # ZeroMQ message, a HTTP request, a HTTP response, etc.
 role Cro::Message {
+    also does Associative;
```

Comment:
Is this necessary?

Reply:
Yes. It's the easiest way to implement what I need.